tests/integration/components/polaris-thumbnail-test.js

```diff
@@ -36,8 +36,7 @@ test('it renders the correct HTML when all attributes are passed in', function(a
 
 
 test('it renders a correctly-sized thumbnail', function(assert) {
-  this.set('size', null);
```

Comment:
Any reason for explicitly getting rid of this `null`?

Reply:
it's kind of redundant.....we pass 'unsupported' at the end and test basically exactly this

Reply:
`null` is different to a string that isn't a member of an enum... 🤷‍♂️ but OK 😛

Reply:
wait...why is it different? it's an enum....`null` like `unsupported` is a value that's not in the enum....shouldn't it be exact same thing?

Reply:
The outside world doesn't know how `size` is being used internally - the component's a black box. Just thought it was worth testing that it doesn't blow up if it's given `null` - imagine if to perform a case-insensitive check against the enum, the component was doing `size.toLowerCase()` for example - `'unsupported'` would be fine, but `null` would blow up